development/resources-cleanup.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
```

Comment:
I would change the name of the script to something more straightforward like `disk-cleanup.sh`. We need to clean multiple resources, so we will probably have a few of scripts like this.

Reply:
Renamed to disks-cleanup.sh